catalogs/aims.yaml

```diff
@@ -1,6 +1,5 @@
 metadata:
   version: 1
-  data_path: aims
```

Comment:
Did you find this causing any problem? I'd like to maintain this for consistency unless we saw an issue with it.

Reply:
No, that was dropped somehow inadvertently